turtle_name: "turtle_leonardo"
pose_x: 0.0
pose_y: 2.0
pose_z: 0.0
pose_rx: 0.0
pose_ry: -30.0
pose_rz: 0.0

